### PR TITLE
fix(schema): add some params to validationSchemaIgnoreParams for better handling

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
will close #1454

File: nextflow.config [hunk 1]
```diff
@@ -97,7 +97,7 @@ params {
     vep_out_format           = "vcf"
     vep_spliceai             = null // spliceai plugin disabled within VEP
     vep_spliceregion         = null // spliceregion plugin disabled within VEP
-    vep_version              = "110.0-0"      // Should be updated when we update VEP, needs this to get full path to some plugins
+    vep_version              = "111.0-0"      // Should be updated when we update VEP, needs this to get full path to some plugins
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply nextflow.config:100]: Changelog?

[the remark this answers is not in the document]
Comment (maxulysse) [reply nextflow.config:100]: yeah, I noticed I forgot about it while I was modifying the schema/config

[the remark this answers is not in the document]
Comment (maxulysse) [reply nextflow.config:100]: CHANGELOG updated

**Comment by asp8200** [comment]
What about the `snpeff_db` thing that Tower was complaining about?  It that also fixed by this PR?

**Comment by maxulysse** [comment]
> What about the `snpeff_db` thing that Tower was complaining about? It that also fixed by this PR?

yeah, that's the initial point of this PR.

**Pull request merged**